skops/hub_utils/_hf_hub.py

```diff
@@ -402,7 +402,14 @@ def update_env(
     -------
     None
     """
-    pass
+
+    with open(Path(path) / "config.json") as f:
+        config = json.load(f)
+
+    config["sklearn"]["environment"] = requirements
+
+    with open(Path(path) / "config.json", mode="w") as f:
+        json.dump(config, f, sort_keys=True, indent=4)
```

Comment:
Just throwing an idea out there: Do we want to factor this line out into a small utility function, since it's used twice now (other being `_create_config`)?

Reply:
Done.